x/twasm/contract/incoming_msgs.go

```diff
@@ -196,6 +196,40 @@ func (p *GovProposal) UnmarshalJSON(b []byte) error {
 			}
 			return nil
 		},
+		"migrate_contract": func(b []byte) error {
+			var proxy = struct { // todo: better use wasmvmtypes.MigrateMsg when names match
```

Comment:
This has a `run_as` parameter as well. See https://github.com/confio/poe-contracts/blob/30972321799e6306a53aaf8ba0d3fb525c74f52f/packages/bindings/src/gov.rs#L57-L66

Reply:
This field was removed: https://github.com/CosmWasm/wasmd/commit/0a96e26fbad8ff67cd130ad151c7f8e968c12d09

Reply:
Will the unmarshall work anyway without that field?

Reply:
See https://github.com/confio/poe-contracts/issues/125